### PR TITLE
TODO: Refactors

### PR DESCRIPTION
**Pull request by FourOFour**
Changing locations of functions at the for best practice and putting them on better location Able to make multiple todo
Accepting config and preset in case of not been there and some minor changes in css and js such as comments and replacing any id with class to be able to have multiple todo without any problems.

### CODE REVIEW

**Comment by orkideh** [comment]
I would suggest to have more descriptive name for your branch and PR title also to have a short bullet points or changes.

File: todo/js/script.js [hunk 1]
```diff
@@ -1,16 +1,49 @@
...
+    */
+    if (config == undefined) {
+        config = {
+            id: 'app',
+            shouldInit: true
+        }
+    }
+    // if Only one was undefined
+    if (config.id == undefined) config.id = 'app';
+    if (config.shouldInit == undefined) config.shouldInit = true;
```
Comment (orkideh) [inline todo/js/script.js:17]: I suggest to use [default value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) and [Object.assign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) to handle default config.

File: todo/js/script.js [hunk 2]
```diff
@@ -270,18 +312,15 @@ const app = new function IIFE() {
...
+    return app;
+}
 
-        return {index, task};
-    }
-}
\ No newline at end of file
+var app = new Todo();
+var app2 = new Todo({id: 'app2', shouldInit: false});
+app2.init();
```
Comment (orkideh) [inline todo/js/script.js:326]: Seems the new at the end of the file is missing

**Comment by orkideh** [comment]
> I would suggest to have more descriptive name for your branch and PR title also to have a short bullet points or changes.

I also suggest a preview link on the PR  bullet points  too

File: todo/js/script.js (hunk 1 above)
Comment (FourOFour) [reply todo/js/script.js:17]: much appreciate the suggestion, what do you think of using `{id = 'app', shouldInit = true} = {}` as the first argument as I had in mind to accept an object for any sort of config?

**Comment by FourOFour** [comment]
- Refactor for best practice
- Some small changes in JS and CSS
- Be able to make multiple todo now

**Comment by FourOFour** [comment]
My apologize, I renamed the branch to 'todo-vanila' and it closed this PR, I will apply the changes and request a new PR, again apologize for inconvenient. @orkideh

**Pull request closed without merging**